ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -111,16 +120,41 @@ TorchCore::~TorchCore ()
   g_free (model_path);
 }
 
+/**
+ * @brief	Set the accelerator for the pytorch
+ */
+void
+TorchCore::setAccelerator (const char *accelerators)
+{
+  use_gpu = TRUE;
+  accelerator = parse_accl_hw (accelerators, torch_accl_support);
+  if (accelerator == ACCL_NONE)
+    goto use_gpu_ini;
+  if ((accelerator & (ACCL_CPU | ACCL_DEFAULT)) != 0)
+    use_gpu = FALSE;
```

Comment:
What does this (Line 134) mean? ```use_gpu``` is in the stack.

Reply:
`use_gpu` is class variable declared on line 73.
accelerator stores the enum value corresponding to the string passed in from the user.
(Line 134) sets `use_gpu` to `FALSE` is string passed in is inferred as `ACCL_DEFAULT` or `ACCL_CPU`.

Reply:
AH.. ok.